src/dash-table/syntax-tree/lexeme/relational.ts

```diff
@@ -73,6 +83,24 @@ export const greaterThan: IUnboundedLexeme = R.merge({
     regexp: /^(>|gt)/i
 }, LEXEME_BASE);
 
+const DATE_OPTIONS: IDateValidation = {
+    allow_YY: true
+};
+
+export const likeDate: IUnboundedLexeme = R.merge({
+    evaluate: relationalEvaluator(([op, exp]) => {
+        const normalizedOp = normalizeDate(op, DATE_OPTIONS);
+        const normalizedExp = normalizeDate(exp, DATE_OPTIONS);
```

Comment:
Try to normalize content with the most flexible configuration (DATE_OPTIONS) possible

Reply:
Otherwise, would need to know the source field + column configuration for a given `evaluate` -- not sure how I'd want to expose that atm